templates/main.php

```diff
@@ -167,46 +167,46 @@ class="collection"
             <div class="app-navigation-entry-menu bubble sortorder-dropdown">
```

Comment:
The click on menu is broken too, this is not just the guidelines 😃 
When you select a sort option, you can't click the button again (or with a lot of difficulties)